public/templates/prWaitingRoom.html

```diff
@@ -104,36 +104,36 @@
     </header>
 
     <h1 id="title">Read Patient Notes, then Join The Meeting!</h1>
-    <div id="pageDiv">
-      <div id="prMain">
+    <div id="page-div">
+      <div id="pr-main">
         <h1 class="heading">
           Join To Meet:
-          <p class="patientInfo" id="patientName">
+          <p class="patient-info" id="patient-name">
```

Comment:
Is there any good reason that the `class` doesn't match the `id`?

Reply:
I believe the "patient-info" class have css that applies to elements on line 118 and 123 as well and "patient-name" has its own specific css

Reply:
This is true. The `patient-info`  class applies to three pieces of information that belong to the patient (i.e. their name, their issue, and the description of their issue). The `patient-name` id apples solely to the first piece of information - the patient's name. The two selectors have their own unique css rules.